rmc/resources/reference_data.py

```diff
@@ -150,18 +157,19 @@
 and phenotypic context of autism (2022)
 """
 
-de_novo = TableResource(
-    path=f"{RESOURCE_BUILD_PREFIX}/reference_data/ht/ddd_autism_de_novo.ht",
+# TODO: Move HT at `ddd_autism_de_novo.ht` to this path
+ndd_de_novo = TableResource(
```

Comment:
Some references to `de_novo` in `calculate_mpc.py` and `constraint.py` have not yet been switched over to `ndd_de_novo`